src/modules/portfolio/components/performance-graph/performance-graph.jsx

```diff
@@ -0,0 +1,277 @@
+          return positions
+        }
+      },
+      plotOptions: {
+        series: {
+          color: 'white',
+          fillColor: {
+            linearGradient: [0, 0, 0, '100%'],
+            stops: [
+              [0, Highcharts.Color('#dbdae1').setOpacity(0.25).get('rgba')],
+              [0.5, Highcharts.Color('#dbdae1').setOpacity(0.15).get('rgba')],
+              [1, Highcharts.Color('#dbdae1').setOpacity(0).get('rgba')]
+            ]
+          }
+        }
+      },
+      legend: {
+        enabled: false
+      },
+      tooltip: {
+        positioner(labelWidth, labelHeight, point) {
+          // console.log(labelWidth, labelHeight, point)
+          return { x: point.plotX - labelWidth, y: point.plotY - (labelHeight * 0.9) }
+        },
+        backgroundColor: 'rgba(255,255,255,0)',
+        borderWidth: 0,
+        headerFormat: '',
+        style: {
+          fontSize: '1rem'
+        },
+        shadow: false,
+        shape: 'none',
+        pointFormat: '<b style="color:{point.color}">{point.y} ETH</b>',
+        valueDecimals: 2
+      },
+      credits: {
+        enabled: false
+      },
+      series: [defaultSeriesData]
+    })
+
+    window.addEventListener('resize', this.updateChartDebounced)
+
+    this.updateChart()
+  }
+
+  componentDidUpdate(prevProps, prevState) {
+    if (prevProps.performanceData !== this.props.performanceData || prevState.graphType !== this.state.graphType || prevState.graphPeriod !== this.state.graphPeriod) this.updateChart()
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('resize', this.updateChartDebounced)
+  }
+
+  changeDropdown(value) {
+    let newType = this.state.graphType
+    let newPeriod = this.state.graphPeriod
+
+    this.state.graphTypeOptions.forEach((type, ind) => {
+      if (type.value === value) {
+        newType = value
+      }
+    })
+
+    this.state.graphPeriodOptions.forEach((period, ind) => {
+      if (period.value === value) {
+        newPeriod = value
+      }
+    })
+
+    this.setState({ graphType: newType, graphPeriod: newPeriod })
+  }
+
+  updateChart() {
+    (getValue(this.props.performanceData, `${this.state.graphType}.${this.state.graphPeriod}`) || []).forEach((series, i) => {
+      if (this.performanceGraph.series[i] == null) {
+        this.performanceGraph.addSeries({
+          type: 'area',
+          color: '#ffffff',
+          className: Styles.PerformanceGraph__Series,
+          marker: {
+            fillColor: 'white',
+            lineColor: 'white',
+          },
+          name: series.name,
+          data: series.data
+        }, false)
+      } else {
+        this.performanceGraph.series[i].setData(series.data, false)
+      }
+    })
+
+    // console.log(this.performanceGraph)
+    this.performanceGraph.redraw()
+  }
+
+  render() {
+    // const p = this.props
+    const s = this.state
+
+    return (
+      <section
+        className={Styles.PerformanceGraph}
```

Comment:
These can be on the same line when only one attribute is being set and it’s not too verbose.  

Style critique mainly.